endpoints/calendar/index.js

```diff
@@ -1,11 +1,57 @@
 import app from '../../server'
 import fridagar from 'fridagar'
 import makeDebug from 'debug'
-import moment from 'moment'
-import { range } from 'lodash'
+import { range, isString } from 'lodash'
 
 const debug = makeDebug('endpoint:calendar')
 
+const canBeInt = intLike => {
+  const num = Number.parseInt(intLike, 10)
+  return !Number.isNaN(num)
+}
+
+export const normalizeParams = (year, month, day) => {
+  // If string parsing failed, reject the promise
+  if (isString(year) && !canBeInt(year)) return { error: 'Year must be a number' }
```

Comment:
Oh no, I just read this again and I think this makes sense. I'm just wondering if this type of homemade validation makes sense. Maybe express can do the parsing and validating for us?